aztec/src/main/kotlin/org/wordpress/aztec/spans/AztecAudioSpan.kt

```diff
@@ -20,6 +20,10 @@ class AztecAudioSpan(context: Context, drawable: Drawable?, override var nesting
     }
 
     override fun onClick() {
-        onAudioTappedListener?.onAudioTapped(attributes)
+        val newAttributes =
```

Comment:
Nit but, can we replace the variable plus null check conditional with:
```suggestion
        onAudioTappedListener?.onAudioTapped(attributes)?.let { attributes = it; }
```

Same for the other spans in this PR.

Reply:
oh thanks, addressed in bb9b116f